src/main/java/seedu/address/logic/commands/ViewStatsCommand.java

```diff
@@ -0,0 +1,21 @@
+package seedu.address.logic.commands;
+
+import seedu.address.logic.CommandHistory;
+import seedu.address.model.Model;
+import seedu.address.ui.StatsWindow;
+
+/**
+ * Clears the address book.
+ */
+public class ViewStatsCommand extends Command {
+
+    public static final String COMMAND_WORD = "stats";
+
+    public static final String SHOWING_STATS_MESSAGE = "Opened stats window.";
```

Comment:
Try to name messages as MESSAGE_.... rather than MESSAGE behind- be consistent with the rest of the app.

Reply:
i followed the format of HelpCommand though